Python/jdbc/read_jdbc.py

```diff
@@ -0,0 +1,93 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# standard libraries
+import logging
+from typing import NamedTuple
+
+# third party libraries
+import apache_beam as beam
+from apache_beam import coders
+from apache_beam import Map
+from apache_beam.io.jdbc import ReadFromJdbc
+from apache_beam.options.pipeline_options import PipelineOptions
+
+
+class ExampleRow(NamedTuple):
+    id: int
+    name: str
+
+
+class JdbcOptions(PipelineOptions):
+    @classmethod
+    def _add_argparse_args(cls, parser):
+        # Add a command line flag to be parsed along
+        # with other normal PipelineOptions
+        parser.add_argument(
+            "--table_name",
+            required=True,
+            help="Table name"
+        )
+        parser.add_argument(
+            "--jdbc_url",
+            required=True,
+            help="JDBC URL"
+        )
+        parser.add_argument(
+            "--driver_class_name",
+            required=True,
+            default="org.postgresql.Driver",
```

Comment:
if default is given, required should be false here.

Reply:
Fixed